src/components/VmUserMessages/index.js

```diff
@@ -3,120 +3,103 @@ import PropTypes from 'prop-types'
 
 import { connect } from 'react-redux'
 
-import style from './style.css'
+import { Notification, NotificationDrawer, MenuItem, Icon, Button } from 'patternfly-react'
 
-import Time from '../Time'
+import style from './style.css'
 
-import { clearUserMessages } from '../../actions/vm'
-import { hrefWithoutHistory } from '../../helpers'
+import { clearUserMessages, displayUserMessages, dismissUserMessage } from '../../actions'
+import { hrefWithoutHistory, getFormatedDateTime } from '../../helpers'
 import { msg } from '../../intl'
 
-const UserMessage = ({ record, id }) => {
-  // TODO: render record.type
-  return (
-    <li className={`list-group-item ${style.crop}`} title={record.message} data-toggle='tooltip'>
-      <span>
-        <pre className={style['message-box']} id={id}>
-          <Time time={record.time} cssClass={style['usermsg-time']} />
-          {record.message}
-        </pre>
-      </span>
-    </li>
-  )
+const UserMessage = ({ record, id, onDismissMessage }) => {
+  const time = getFormatedDateTime(record.get('time'))
+  return (<Notification seen>
```

Comment:
Where is `seen` defined?

Reply:
There is not need to define `seen`, enough just add it as props.
P.S. Best practice for set bool props.

Reply:
Ok, I got confused. You're just rendering all Notifications as seen.

At first glance, I assumed you intended to keep seen vs unseen (read vs unread).